apps/marketplace/components/BuyerSpecialist/BuyerSpecialistSelectStage.js

```diff
@@ -89,9 +87,9 @@ export class BuyerSpecialistSelectStage extends Component {
         model={this.props.model}
         validators={{
           '': {
-            requiredCategory,
             requiredChoice,
-            atLeastOneSeller
+            atLeastOneSeller,
+            requiredCategory
```

Comment:
This is another example of code that didn't have to change.

Reply:
True, it didn't need to change. Swapping the order of the validators shows error messages in the same order as the seller & category fields, which is only apparent if the user does not complete either field (likely a very rare scenario).